tests/test_enum_field.py

```diff
@@ -250,8 +250,8 @@ class TestLoadDumpConfigBehavior(object):
 
     def test_load_and_dump_by_default_to_by_value(self):
```

Comment:
its kind of funny that even the test name is `test_load_and_dump_by_default_to_by_value` but for some reason the default is name, not value